db/transaction_log_impl.h

```diff
@@ -54,14 +55,68 @@ class LogFileImpl : public LogFile {
   uint64_t sizeFileBytes_;
 };
 
+class TransactionLogSeqCache {
+ public:
+  TransactionLogSeqCache(size_t size)
+      : size_(size), rand_(std::random_device{}()) {}
+  struct SeqWithFileBlockIdx {
+    uint64_t log_number;
+    uint64_t seq_number;
+    uint64_t block_index;
+    SeqWithFileBlockIdx(uint64_t log_num, uint64_t seq_num, uint64_t block_idx)
+        : log_number(log_num), seq_number(seq_num), block_index(block_idx){};
+    bool operator<(const SeqWithFileBlockIdx& other) const {
```

Comment:
Would block_index be different with the same log_number and seq_number?

Reply:
There shouldn't be such a situation, but retaining the judgment for block_idx seems no problem